docs/modules/user-guide/partials/proc_adding-kubernetes-component-to-a-devfile.adoc

```diff
@@ -12,116 +12,45 @@ This section describes how to add a `kubernetes` component to a devfile. A compl
 
 .Procedure
 
-. Define a component using the type `kubernetes`.
+. Define a component using the type `kubernetes` or `openshift`.
 
-. The content can be provided through the `reference` attribute, which points to the file with the component content.
+. Provide the content through the `uri` or `inlined` property.
```

Comment:
We have an example showing use of the `uri` property. How is `inline` used? I can infer its usage, but it might be a good idea to make it explicit to the reader that the contents of the property are specified directly in the YAML as a string.

Reply:
Will address this concern in a PR fo the newly filed issue, #393: devfile/api#393